### PR TITLE
Add setup script for production.

### PR DESCRIPTION
**Pull request by LyonesGamer**
Script will install dependencies, run migrations, and create root account.

### CODE REVIEW

**Comment by Kennric** [comment]
+1

**Comment by ElijahCaine** [comment]
+1

File: knexfile.js [hunk 1]
```diff
@@ -1,30 +1,44 @@
...
+    connection: process.env.PG_CONNECTION_STRING,
+  },
+
+  production_mysql: {
+    client: 'mysql',
+    connection: process.env.MYSQL_CONNECTION_STRING,
+  },
+
+  production_sqlite: {
+    client: 'mysql',
```
Comment (athai) [inline knexfile.js]: Did you mean 'sqlite'?

File: setup.sh [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    export MYSQL_CONNECTION_STRING="mysql://${MYSQL_USER}:${MYSQL_PASSWORD}@${MYSQL_HOST}:${MYSQL_PORT}/${MYSQL_DB}"
+    export NODE_ENV="production_mysql"
+  elif [ "$DATABASE" == "postgresql" -o "$DATABASE" == "postgres" -o "$DATABASE" == "pg" ]; then
+    read -p "PostgreSQL user: " PG_USER
+    read -p "PostgreSQL password: " PG_PASSWORD
+    read -p "PostgreSQL hostname: " PG_HOST
+    read -p "PostgreSQL port: " PG_PORT
+    read -p "PostgreSQL database: " PG_DB
+    export PG_CONNECTION_STRING="postgres://${PG_USER}:${PG_PASSWORD}@${PG_HOST}:${PG_PORT}/${PG_DB}"
+    export NODE_ENV="production_pg"
```
Comment (athai) [inline setup.sh]: I'm curious - why it "production_pg" encased in double quotes here, but not below (on line 48)?

**Pull request merged**